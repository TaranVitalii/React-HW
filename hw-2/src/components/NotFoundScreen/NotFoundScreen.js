import React from "react";

export default () => {
  return <div>User Not a found</div>;
};
