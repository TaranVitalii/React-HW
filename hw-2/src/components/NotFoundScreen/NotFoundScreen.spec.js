import React from "react";
import NotFoundScreen from "./NotFoundScreen";
import { shallow } from "enzyme";

describe("<NotFoundScreen/>", () => {
  it("should render a not found screen", () => {
    const wrapper = shallow(<NotFoundScreen />);

    expect(wrapper).toMatchSnapshot();
  });
});
