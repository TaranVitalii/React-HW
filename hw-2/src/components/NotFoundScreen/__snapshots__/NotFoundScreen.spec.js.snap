// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputForm/> 1st test 1`] = `ShallowWrapper {}`;

exports[`<NotFoundScreen/> should render a not found screen 1`] = `ShallowWrapper {}`;
